tests/unit/lib/test_signal_schema.py

```diff
@@ -356,6 +357,24 @@ def test_slice_nested():
     assert list(sliced.values.items()) == [("feature.aa", int)]
 
 
+def test_mutate_rename():
```

Comment:
are these tests related?

Reply:
No, I just realized we are missing tests for that one so added them. They are pretty simple ones so I didn't want to open a new PR just for it.